### DISCUSSION TITLE
E ink application launcher for android.

### DISCUSSION THREAD

**Discussion by Zeyadas** [koreader/koreader#7454] [Q&A]
I was wondering if there's something like (KUAL: Kindle Unified Application Launcher) but for Android?
I mean I need a launcher that work on android and make it behave as a dedicated e ink reader.
I found some (actually two) apps:
This one and I couldn't download it I don't know why.. 
https://bitbucket.org/dsimbiriatin/simple-ink-launcher/wiki/Home
And the other one is this:
https://m.apkhere.com/app/cn.modificator.launcher and it's not (perfect).

I don't know if I described it clearly..

**Comment by Frenzie** [comment]
There's a variety of launchers that don't depend on movement, e.g., https://github.com/SubhamTyagi/Last-Launcher

That being said, it seems that your regular icon-based launcher should be fine too as long as you avoid the animation-based stuff.

**Reply by Zeyadas** [reply]
That was helpful, thank you..
Any other suggestions (launchers)?

**Reply by Frenzie** [reply]
Not off the top of my head.
